hoth_api.yaml

```diff
@@ -299,11 +298,21 @@ components:
         - operators
       properties:
         operators:
+          description: Array of operator objects
           type: array
-          description: Array of operator email addresses
           items:
-            type: string
-            example: "user@mail.mil"
+            $ref: "#/components/schemas/Operator"
+    Operator:
+      description: Operator data for provisioning resources in a CSP
+      type: object
+      properties:
+        email:
+          type: string
+          example: "user@mail.mil"
+        dod_id:
+          type: string
+        needs_reset:
+          type: boolean
```

Comment:
Are none of these required?

Reply:
In the database schema I marked email and dod_id as mandatory, so perhaps do the same here

Reply:
oh yeah, they are I'll fix that. Thanks! 👍

Reply:
@jasonburkert-ccpo I made `needsReset` required too since version `0.3.0` of ATAT schema has it with a default of `false`.

Reply:
Added in https://github.com/dod-ccpo/atat-web-api/pull/1098/commits/778c44a2da8872ab06991c8e176ab4112c552b96